
ERROR_TIME: 2021-08-23 16:18:52,943
                        
FILE_NAME: teki_main_app.py
                        
MODULE: teki_main_app
                        
LINE_NO: 1383
ERROR_SCOPE  'if __name__ == __main__': is due to 'name 'bs4' is not defined)'
Traceback (most recent call last):
  File "C:\Users\chris\Desktop\B.A\Teki\teki_main_app.py", line 1366, in <module>
    run_program(default_doc, default_train, system_evaluation)
  File "C:\Users\chris\Desktop\B.A\Teki\teki_main_app.py", line 1236, in run_program
    doc = get_text(default_doc)
  File "C:\Users\chris\Desktop\B.A\Teki\teki_main_app.py", line 221, in get_text
    soup = bs4.BeautifulSoup(file, "lxml")
NameError: name 'bs4' is not defined

ERROR_TIME: 2021-08-23 16:21:27,921
                        
FILE_NAME: teki_main_app.py
                        
MODULE: teki_main_app
                        
LINE_NO: 1383
ERROR_SCOPE  'if __name__ == __main__': is due to 'name 'bs4' is not defined)'
Traceback (most recent call last):
  File "C:\Users\chris\Desktop\B.A\Teki\teki_main_app.py", line 1379, in <module>
    run_program(default_doc, default_train, system_evaluation)
  File "C:\Users\chris\Desktop\B.A\Teki\teki_main_app.py", line 1236, in run_program
    doc = get_text(default_doc)
  File "C:\Users\chris\Desktop\B.A\Teki\teki_main_app.py", line 221, in get_text
    soup = bs4.BeautifulSoup(file, "lxml")
NameError: name 'bs4' is not defined
